packages/node_modules/@webex/internal-plugin-conversation/src/conversation.js

```diff
@@ -363,11 +363,23 @@ const Conversation = WebexPlugin.extend({
       return Promise.reject(new Error('`object` must be an object'));
     }
 
-    return this.prepare(activity, {
+    const request = {
       verb: 'delete',
       target: this.prepareConversation(convoWithUrl),
-      object: pick(object, 'id', 'url', 'objectType')
-    })
+      object: pick(object, 'id', 'url', 'objectType'),
+    };
+
+    // Deleting meeting container requires KMS message
+    if (object.object.objectType === 'meetingContainer') {
+      // It's building a string resourceUri + "/authorizations?authId=" + id, where resourceUri is the meeting container's KRO URL, and id is the conversation's KRO URL.
+      request.kmsMessage = {
+        method: 'delete',
+        uri: `${object.object.kmsResourceObjectUrl}/authorizations?${querystring.stringify({authId: convoWithUrl.kmsResourceObjectUrl})}`
```

Comment:
question: Your comment mentions `resourceUri` (and the prepareActivityKmsMessage function uses both uri and resourceUri) but are creating `uri`. Are they both the meant to be same? Is one no longer required?

Reply:
I've seen it with either one, however for this case, it has to be uri